docs/reference/config-params.md

```diff
@@ -280,462 +324,543 @@ Otherwise it will default to `1`.
 
 ### Possible options: Arbitrary integer value. Example: `10`.
 
-> kev.workload.rolling-update-max-surge:
+> x-k8s.workload.rollingUpdateMaxSurge:
 ```yaml
 version: 3.7
 services:
   my-service:
-    labels:
-      kev.workload.rolling-update-max-surge: 2
+    x-k8s:
+      workload:
+        rollingUpdateMaxSurge: 2
 ...
 ```
 
-## kev.workload.cpu
+## x-k8s.workload.resource
+
+Defines the resource share request for a given workload using different parameters.
+
+### x-k8s.workload.resource.cpu
 
 Defines the CPU share request for a given workload. See official K8s [documentation](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/). Kev will attempt to infer CPU request from the information specified in the compose file.
 
-Kev uses the following heuristics to derieve that information for each service:
+Kev uses the following heuristics to derive that information for each service:
 
 If compose file(s) specifies the `deploy.resources.reservations.cpus` attribute key in a project service config it will use its value. Otherwise it'll assume sensible default of `0.1` (equivalent of 100m in Kubernetes).
 
-### Default: `0.1`
+#### Default: `0.1`
 
-### Possible options: Arbitrary [CPU units](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#meaning-of-cpu). Examples: `0.2` == `200m`.
+#### Possible options: Arbitrary [CPU units](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#meaning-of-cpu). Examples: `0.2` == `200m`.
 
-> kev.workload.cpu:
+> x-k8s.workload.resource.cpu:
 ```yaml
 version: 3.7
 services:
   my-service:
-    labels:
-      kev.workload.cpu: 1
+    x-k8s:
+      workload:
+        resource:
+          cpu: 0.1
 ...
 ```
 
-## kev.workload.max-cpu
+### x-k8s.workload.resource.maxCpu
 
-Defines the CPU share limit for a given workload. See official K8s [documentation](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/). Kev will attempt to infer CPU request from the information specified in the compose file.
+Defines the max CPU share limit for a given workload. See official K8s [documentation](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/). Kev will attempt to infer CPU request from the information specified in the compose file.
```

Comment:
I think the `limit` sets the boundaries here so probably can omit added `max` here?